src/js/parsers/captions/captions.d.ts

```diff
@@ -0,0 +1,5 @@
+export type CaptionEntryData = {
```

Comment:
made this shared file for all types captions (given this is shared between a bunch of parsers)